### ISSUE TITLE
The consistent of hyperparameter ctc_weight and mtlalpha

### ISSUE THREAD

**Issue by mpc001** [espnet/espnet#254]
Hi,
    mtalpha in loss function and ctc_weight in joint-decoding use the same notation according to the tookit paper. Does that mean these two hyperparameters should be same? I notice script in voxforge and hkust recipe used distinct values for these two parameters. Many thanks in advance!

**Comment by sw005320** [comment]
This does not have to be same, although the paper uses the same notation. Thanks for your clarification. I'll revise the arxiv paper.

**Comment by mpc001** [comment]
Hi, @sw005320 
Thanks for your quick reply!
